commit a05f7bad3f07a4041e7011a629858a8b0921e9e0
Author: Anatol Pomozov <anatol.pomozov@gmail.com>
Date:   Fri May 15 20:04:43 2020 -0700

    Do not use third-party dependency as it is missing in Android's version of the repo

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 75bf9981..9ec8e81b 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -562,13 +562,6 @@ if(USE_CUSTOM_LIBCXX)
   target_link_libraries(libcxx libcxxabi)
 endif()
 
-# Add minimal googletest targets. The provided one has many side-effects, and
-# googletest has a very straightforward build.
-add_library(boringssl_gtest third_party/googletest/src/gtest-all.cc)
-target_include_directories(boringssl_gtest PRIVATE third_party/googletest)
-
-include_directories(third_party/googletest/include)
-
 # Declare a dummy target to build all unit tests. Test targets should inject
 # themselves as dependencies next to the target definition.
 add_custom_target(all_tests)
